src/shogun/machine/gp/FITCInferenceMethod.cpp

```diff
@@ -246,6 +228,7 @@ void CFITCInferenceMethod::update_chol()
 	m_L=SGMatrix<float64_t>(m_kuu.num_rows, m_kuu.num_cols);
 	Map<MatrixXd> eigen_chol(m_L.matrix, m_L.num_rows, m_L.num_cols);
 
+	//post.L  = solve_chol(Lu*Luu,eye(nu)) - iKuu;
```

Comment:
@lambday we need something as
http://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.cho_solve.html

or at least a triangular solve in linalg